api_tests/wb/views/test_wb_hooks.py

```diff
@@ -802,6 +840,43 @@ def test_invalid_payload(self, app, copy_url):
         assert res.status_code == 400
         assert res.json['errors'][0]['detail'] == 'Invalid Payload'
 
+    def test_storage_usage_move_within_node(self, app, node, signed_payload, copy_url):
```

Comment:
Test name is a little confusing, can you change so it references a "copy" instead of a "move"